ui-performance/index.md

```diff
@@ -121,27 +121,51 @@ Gallery example:
 <center>The graphs produced by the performance overlay.<br>The vertical green
 bars represent the current frame.</center><br>
 
-Each graph represents the last 300 frames for the following two threads used
-in every Flutter app:
+Flutter uses several threads to do its work. You have direct control
+over what happens in the UI thread, but no direct access to the other
+threads.
+
+1. UI thread<br>
+   The platform's main thread executes Dart code in the Dart VM.
+   This thread includes code that you wrote, and code executed by
+   Flutter's framework on your app's behalf.
+   Also referred to as Flutter's _engine_, plugins typically run here.
```

Comment:
This is incorrect. Plugin code runs on the "Platform" thread. Dart code runs on the "UI" thread.